### PR TITLE
[Merged by Bors] - chore: adaptations for nightly-2024-09-04

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Algebra/BigOperators/Finsupp.lean [hunk 1]
```diff
@@ -97,7 +97,7 @@ theorem sum_ite_self_eq [DecidableEq α] {N : Type*} [AddCommMonoid N] (f : α 
     (f.sum fun x v => ite (a = x) v 0) = f a := by
   classical
     convert f.sum_ite_eq a fun _ => id
-    simp [ite_eq_right_iff.2 Eq.symm]
+    simp [(ite_eq_right_iff _ _ _).2 Eq.symm]
```
Comment (semorrison) [inline Mathlib/Algebra/BigOperators/Finsupp.lean:100]: Apologies for these changes, they were a mistake and will be reverted back again...

File: Mathlib/Data/Nat/Bits.lean [hunk 2]
```diff
@@ -48,7 +48,7 @@ def bodd (n : ℕ) : Bool := (boddDiv2 n).1
 
 @[simp] lemma bodd_zero : bodd 0 = false := rfl
 
-lemma bodd_one : bodd 1 = true := rfl
+@[simp]lemma bodd_one : bodd 1 = true := rfl
```
Comment (Ruben-VandeVelde) [inline Mathlib/Data/Nat/Bits.lean:51]:
```suggestion
@[simp] lemma bodd_one : bodd 1 = true := rfl
```

**Comment by Ruben-VandeVelde** [comment]
maintainer merge

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

**Pull request closed without merging**